pages/wishListEmpty.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://dash-css.netlify.app/css/style.css">
+    <link rel="stylesheet" href="../reset.css">
+    <link rel="stylesheet" href="../app.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="nav_left_side flex flex-center gap-s">
+            <a class="link_wrapper" href="../pages/home.html">
+                <i class="fas fa-axe-battle fa-3x"></i>
+                <span class="h3">G-Commerce</span>
+            </a>
+        </div>
+        <div class="nav_search">
+            <input class="input_regular input_round" placeholder="Search">
+            <button class="btn btn_light">Search</button>
+        </div>
+        <div class="nav_right_side">
+            <a class="link_wrapper" href="../pages/signIn.html">
+                <button class="btn btn_light">Log In</button>
+            </a>
+            <a class="link_wrapper" href="../pages/wishlist.html">
+                <span class="badge_wrapper">
+                    <i class="fas fa-heart fa-2x"></i>
+                    <span class="badge badge_circle badge_primary position_absolute right-0 top-0">4</span>
+                </span>
+            </a>
+            <a class="link_wrapper" href="../pages/cart.html">
+                <span class="badge_wrapper">
+                    <i class="fas fa-shopping-cart fa-2x"></i>
+                    <span class="badge badge_circle badge_primary position_absolute right-0 top-0">4</span>
+                </span>
+            </a>
+        </div>
+
+    </nav>
+    <div class="main_container flex flex-col flex-center py-small">
+        <div class="h3">YOUR WISHLIST IS EMPTY</div>
+        <div class="h5 color_grey_200">Add items that you like to your wishlist. Review them anytime and easily move
+            them to the bag.</div>
+        <img class="empty_image" src="../assets/empty_wishlist.png" alt="EMPTY">
```

Comment:
Should Practice having / before closing img tag

Reply:
Thanks for letting me know.Fixed it.